trustfall_core/src/ir/indexed.rs

```diff
@@ -29,8 +29,6 @@ pub struct IndexedQuery {
 pub struct Output {
     pub name: Arc<str>,
 
-    #[serde(serialize_with = "crate::ir::serialization::serde_type_serializer")]
```

Comment:
We don't necessarily have to get rid of these, but I don't think it's good to keep them if we already have the serialize and deserialize directly on `Type`

Reply:
It's a breaking change for any users, but we now implement this directly on our `Type` struct so It's probably fine.